Wind box added

rwchaneyjr opened the pull request:
There is a windbox which was added it is a box which surrounds the scene when an object (ie yarn) collides with it it is given a direction and a force to go like thewind is affecting it.

## Changes
<!-- Provide a list of high level changes for your work on this Pull Request. -->

### Known Bugs/Issues

## Testing Scene and Script
<!--- Provide the name of the scene(s) and scripts that the reviewer needs to check to make sure your task is finished -->

## Issue ticket number/link
<!--- Provide a link or ticket number for an associated issue -->

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Make sure I am merging my features into the `develop` branch
- [ ] I have notified the other members of the programming team that my task is ready to review

rwchaneyjr committed:
Wind box added

There is a windbox which was added it is a box which surrounds the scene when an object (ie yarn) collides with it it is given a direction and a force to go like thewind is affecting it.

rwchaneyjr committed:
Windtunnel

The wind tunnel and the fan is in the game.

SurajSSingh reviewed (changes requested):
Tested in robertPrototypeYarnDamage scene, don't forget to make the box collider a trigger. When made into trigger, force is working, but force angle goes in wrong direction (upwards instead of forward in relation to the fan). May just need to change the force direction in the code (could also expose it in the inspector to allow changes in the inspector).

Also, you committed project version change, you can remove that before merging it in. We can make a separate change for version updates

rwchaneyjr committed:
changes

SurajSSingh commented on Assets/Scenes/robertPrototypeYarnDamage.unity:
```diff
@@ -0,0 +1,3690 @@
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 1666228381}
+  m_Material: {fileID: 0}
+  m_IsTrigger: 0
```
Don't forget to set the Wind Box's box collider to trigger so that the wind script works.

rwchaneyjr replied:
I put the function as OnTriggerStay(Collider other){  }

SurajSSingh commented on Assets/Scripts/Ball/Wind.cs:
```diff
@@ -0,0 +1,19 @@
+        if (hitObj != null)
+        {
+            var rb = hitObj.GetComponent<Rigidbody>();
+            var dir = transform.forward;
```
You will want to promote this to a `[SerializedField]` field at the top so we can control it in the inspector. Seems like I made a mistake in direction, so having this be in the inspector would have made it easier to fix.

SurajSSingh reviewed (changes requested):
Just two small things now to fix:
1. Set Box Collider for Wind Box to trigger,
2. Make wind direction assignable in the inspector

rwchaneyjr commented:
I put the boolean variable up in the  inspector as public. If you click it you choose the .up instead of .forward on the for the force to be applied on the vector.

rwchaneyjr commented:
I put the public boolean variable up in the inspector which when clicked it uses .up for the vector instead of .forward giving the vecotor a force

rwchaneyjr committed:
4 directions added and trigger

the 4 directions left right forward and up were exposed in the inspector to give a force to the yarn ball

rwchaneyjr committed:
updated windbox

SurajSSingh committed:
Reverse version and package changes

Will be split to a different PR

SurajSSingh committed:
Revert update to EditorSettings.asset

Reverse small change to AsyncShaderCompilation
